include/openthread-windows-config.h

```diff
@@ -70,6 +70,9 @@
 #define snprintf sprintf_s
 #endif // WINDOWS_KERNEL
 
+// Redefine rand to random for test code
```

Comment:
Code looks like the other way around from comment: "Redefine random to rand for test code"